DFC.ServiceTaxonomy.ContentApproval/Handlers/ContentApprovalAuditTrailEventHandler.cs

```diff
@@ -105,19 +106,31 @@ public override Task CreateAsync(AuditTrailCreateContext context)
                                         break;
 
                                     case "submit.RequestApproval-ContentDesign":
-                                        contentEvent.Name = "Requested for review (ContentDesign)";
+                                        reviewType = "content design";
+                                        contentEvent.Name = contentApprovalPart.ReviewStatus != Models.Enums.ReviewStatus.InReview
```

Comment:
Could we extract these 3 lines to a local method and call it SetContentEventName or something? This is being used in 4 places.

Reply:
Or rather GetContentEventName, so you do the assignment here by just getting the value from that method.

Reply:
Added GetContentEventName() method

Reply:
Removed ContentEvent parameter.